docs/resources/tool_comparison.md

```diff
@@ -73,4 +73,4 @@ Ray Datasets make it easy to feed data really efficiently into Ray's model train
 
 However, Ray Datasets are not a fully-fledged Dataframe abstraction (and [it is explicit in not being an ETL framework for data science](https://docs.ray.io/en/latest/data/data.html)) which means that it lacks key features in data querying, visualization and aggregations.
 
-Instead, Ray Data is a perfect destination for processed data from DaFt Dataframes to be sent to with a simple [`df.to_ray_dataset()`][daft.DataFrame.to_ray_dataset] call. This is useful as an entrypoint into your model training and inference ecosystem!
+Instead, Ray Data is a perfect destination for processed data from Daft's Dataframes to be sent to with a simple [`df.to_ray_dataset()`][daft.DataFrame.to_ray_dataset] call. This is useful as an entrypoint into your model training and inference ecosystem!
```

Comment:
"from Daft to be sent to"

TBH this sentence is kind of badly structured, you could consider reworking it too.